src/app/SIPUserAgents/SIPCallDescriptor.cs

```diff
@@ -108,6 +108,10 @@ public class SIPCallDescriptor
         public Guid DialPlanContextID;
         public int ReinviteDelay = -1;          // If >= 0 a SIP re-INVITE request will be sent to the remote caller after this many seconds. This is an attempt to work around a bug with one way audio and early media on a particular SIP server.
 
+        //rj2
+        public string CallId;
+        public string BranchId;
+

```

Comment:
Need a comment here warning that this is not recommended and an example of the rare case where it's useful. In particular setting the BranchId can result in unexpected behaviour. It the same value is used for multiple calls they may get associated to the same tx and weird things will happen.